### PR TITLE
Direct frelax solve

### PR DESCRIPTION
**Pull request by oseikuffuor1**
This PR brings in additional options to use Gaussian-Elimination as options for F-relaxation for MGR. The options supported are gselim options 9, 99, and 199.

### CODE REVIEW

File: src/parcsr_ls/par_mgr_setup.c [hunk 1]
```diff
@@ -1925,3 +1968,88 @@ hypre_MGRSetupFrelaxVcycleData( void *mgr_vdata,
 
    return hypre_error_flag;
 }
+#if 0
```
Comment (victorapm) [inline src/parcsr_ls/par_mgr_setup.c]: @oseikuffuor1 Do we want to keep this code?

**Review by victorapm — APPROVED** [review]
Thanks, Daniel! Looks good!

**Review by rfalgout — APPROVED** [review]
Looks good to me.

File: src/parcsr_ls/par_mgr_setup.c (hunk 1 above)
Comment (oseikuffuor1) [reply src/parcsr_ls/par_mgr_setup.c]: @victorapm this section has been removed. Thanks.

**Pull request merged**